README.md

```diff
@@ -215,28 +212,49 @@ import 'package:flutter_credit_card/flutter_credit_card.dart';
       },
       autovalidateMode: AutovalidateMode.always,
       disableCardNumberAutoFillHints: false,
-      cardNumberDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'Number',
-        hintText: 'XXXX XXXX XXXX XXXX',
-      ),
-      expiryDateDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'Expired Date',
-        hintText: 'XX/XX',
-      ),
-      cvvCodeDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'CVV',
-        hintText: 'XXX',
-      ),
-      cardHolderDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'Card Holder',
+      inputConfiguration: const InputConfiguration(
+        cardNumberDecoration: const InputDecoration(
+          border: OutlineInputBorder(),
+          labelText: 'Number',
+          hintText: 'XXXX XXXX XXXX XXXX',
+        ),
+        expiryDateDecoration: const InputDecoration(
+          border: OutlineInputBorder(),
+          labelText: 'Expired Date',
+          hintText: 'XX/XX',
+        ),
+        cvvCodeDecoration: const InputDecoration(
```

Comment:
remove `const` from `InputDecoration` widget as it is specified at the parent constructor and thus is not needed.